src/harlequin_sqlite/adapter.py

```diff
@@ -279,6 +280,10 @@ def __init__(
             self.can_load_extensions = hasattr(
                 sqlite3.Connection, "enable_load_extension"
             )
+            self.ADAPTER_DETAILS = (
```

Comment:
This makes me think a class var isn't the right way to do this -- I think Harlequin should just present the configured values for all of the `ADAPTER_OPTIONS`; that way adapter authors don't have to implement anything.